contracts/ETO/ETOTokenTerms.sol

```diff
@@ -85,14 +88,19 @@ contract ETOTokenTerms is Math, IContractId {
         EQUITY_TOKEN_NAME = equityTokenName;
         EQUITY_TOKEN_SYMBOL = equityTokenSymbol;
         EQUITY_TOKENS_PER_SHARE = equityTokensPerShare;
```

Comment:
we dont mean ULPS here, do we?

like this: i'd read it as EQUITY_TOKENS_PER_SHARE = 10 && PRECISION = 2  => 1 Share = 1 * 10 * 10^2 = 1000 Tokens